src/nft/components/icons.tsx

```diff
@@ -1299,8 +1292,8 @@ export const PlusIconLarge = (props: SVGProps) => (
 )
 
 export const ChevronLeftIcon = (props: SVGProps) => (
-  <svg fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
-    <path d="M15 6L9 12L15 18" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" />
+  <svg width="8" height="16" viewBox="0 0 8 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
i think we these are supposed to be passed in as props.

Reply:
these are defaults pulled from the Figma. Height and Width passed in via props would overwrite

Reply:
<img width="472" alt="Screen Shot 2022-08-24 at 10 00 55 AM" src="https://user-images.githubusercontent.com/4899429/186438674-29191aa1-bc06-4ff4-a050-cfaa95bcc7b8.png">
<img width="322" alt="Screen Shot 2022-08-24 at 10 02 19 AM" src="https://user-images.githubusercontent.com/4899429/186438675-bc9d3007-659f-49e7-9265-71548502bb7b.png">

In this case, it's not an issue since only used in one place, but `{...props}` won't override `height=` in the component. you'd need to explicitly destructure the prop and set a default on it.